frontend/src/components/create-user-page/CreateUser.tsx

```diff
@@ -61,49 +98,81 @@ export const CreateUser: React.FunctionComponent = () => {
     }
   };
 
   return (
-    <div>
+    <div className="form">
       <form onSubmit={handleSubmit}>
-        <label>
-          Name:
-          <input
-            type="text"
-            name="Name"
-            onChange={(e) => setFormName(e.target.value)}
-          />
-        </label>
-        <label>
-          Username:
-          <input
-            type="text"
-            name="Username"
-            onChange={(e) => setFormUsername(e.target.value)}
-          />
-        </label>
-        <label>
-          Email:
-          <input
-            type="email"
-            name="Email"
-            onChange={(e) => setFormEmail(e.target.value)}
-          />
-        </label>
-        <label>
-          Password:
-          <input
-            type="password"
-            name="Password"
-            onChange={(e) => setFormPassword(e.target.value)}
-            onBlur={(e) => validatePassword(e.target.value)}
-          />
-        </label>
-        <input type="submit" value="Create User" />
+        <h1 className="form_heading">Create New User</h1>
+        <div className="form_field">
+          <div className="form_field--row">
+            <label className="form_field--label">Name:</label>
+            <input
+              className="form_field--input"
+              type="text"
+              name="Name"
+              onChange={(e) => validateName(e.target.value)}
+            />
+          </div>
+          <p className="message">{nameMessage}</p>
+        </div>
+        <div className="form_field">
+          <div className="form_field--row">
+            <label className="form_field--label">Username:</label>
+            <input
+              className="form_field--input"
+              type="text"
+              name="Username"
+              onChange={(e) => validateUsername(e.target.value)}
+            />
+          </div>
+          <p className="message">{usernameMessage}</p>
+        </div>
+        <div className="form_field">
+          <div className="form_field--row">
+            <label className="form_field--label">Email:</label>
+            <input
+              className="form_field--input"
+              type="email"
+              name="Email"
+              onChange={(e) => validateEmail(e.target.value)}
+            />
+          </div>
+          <p className="message">{emailMessage}</p>
+        </div>
+        <div className="form_field">
+          <div className="form_field--row">
+            <label className="form_field--label">Password:</label>
+            <input
+              className="form_field--input"
+              type="password"
+              name="Password"
+              onBlur={(e) => validatePassword(e.target.value)}
+            />
+          </div>
+          <p className="message">{passwordMessage}</p>
+        </div>
+        <div className="form_button">
+          <input type="submit" value="Create User" />
+        </div>
       </form>
+      {addUserMessage}
```

Comment:
Some styling on here maybe? At minimum this should be inside a `<p>`

Reply:
done, thanks